pkg/config/config.go

```diff
@@ -38,13 +38,14 @@ type JSONPath struct {
 
 // Task this is the actual task which can be triggered within an Action
 type Task struct {
-	Name      string     `yaml:"name"`
-	Files     []string   `yaml:"files"`
-	Image     string     `yaml:"image"`
-	Cmd       []string   `yaml:"cmd"`
-	Args      []string   `yaml:"args"`
-	Env       []Env      `yaml:"env"`
-	Resources *Resources `yaml:"resources"`
+	Name       string     `yaml:"name"`
+	Files      []string   `yaml:"files"`
+	Image      string     `yaml:"image"`
+	Cmd        []string   `yaml:"cmd"`
+	Args       []string   `yaml:"args"`
+	Env        []Env      `yaml:"env"`
+	Resources  *Resources `yaml:"resources"`
+	WorkingDir string     `yaml:"workingDir"`
```

Comment:
I guess extending one of the configs in [config_test.go](https://github.com/keptn-sandbox/job-executor-service/blob/main/pkg/config/config_test.go) just to see if unmarshalling works without errors would be nice.